src/main/core-api/java/com/mysql/cj/conf/PropertyDefinitions.java

```diff
@@ -713,6 +713,9 @@ public enum DatabaseTerm {
                 new BooleanPropertyDefinition(PropertyKey.useAwsIam, DEFAULT_VALUE_FALSE, RUNTIME_NOT_MODIFIABLE,
                         Messages.getString("ConnectionProperties.useAwsIam"), "0.3.0", CATEGORY_SECURITY, Integer.MAX_VALUE),
 
+                new BooleanPropertyDefinition(PropertyKey.loadBalanceReadOnlyTraffic, DEFAULT_VALUE_FALSE, RUNTIME_NOT_MODIFIABLE,
+                        Messages.getString("ConnectionProperties.loadBalanceReadOnlyTraffic"), "1.2.0", CATEGORY_HA, Integer.MAX_VALUE),
```

Comment:
Since the Read-write splitting plugin is a whole new feature, would the new release be v2.0.0? @hsuamz

Reply:
Chatted with Matthew and he mentioned 1.2.0 is the correct version